src/main/kotlin/com/openlattice/hazelcast/serializers/OrganizationExternalDatabaseColumnStreamSerializer.kt

```diff
@@ -0,0 +1,57 @@
+package com.openlattice.hazelcast.serializers
+
+import com.hazelcast.nio.ObjectDataInput
+import com.hazelcast.nio.ObjectDataOutput
+import com.kryptnostic.rhizome.pods.hazelcast.SelfRegisteringStreamSerializer
+import com.openlattice.hazelcast.StreamSerializerTypeIds
+import com.openlattice.organization.OrganizationExternalDatabaseColumn
+import org.springframework.stereotype.Component
+import java.util.*
+
+@Component
+class OrganizationExternalDatabaseColumnStreamSerializer : SelfRegisteringStreamSerializer<OrganizationExternalDatabaseColumn> {
+
+    companion object {
+        fun serialize(output: ObjectDataOutput, obj: OrganizationExternalDatabaseColumn) {
+            UUIDStreamSerializer.serialize(output, obj.id)
+            output.writeUTF(obj.name)
+            output.writeUTF(obj.title)
+            output.writeUTF(obj.description)
+            UUIDStreamSerializer.serialize(output, obj.tableId)
+            UUIDStreamSerializer.serialize(output, obj.organizationId)
+            output.writeUTF(obj.dataType)
```

Comment:
what is the datatype of datatype? If it's a string, we should probably convert to an enum

Reply:
It is a string. It's whatever the datatype is in postgres. I can make it an enum class if that would be better.